pkg/quick/quick_test.go

```diff
@@ -240,28 +304,42 @@ func (s *MySuite) TestJSONFormat(c *C) {
 
 	// Save format using
 	config, err := New(&saveMe)
-	c.Assert(err, IsNil)
-	c.Assert(config, Not(IsNil))
+	if err != nil {
+		t.Fatal(err)
+	}
 
```

Comment:
Similar comment to the previous one. Is it OK, to not check for config being not nil

Reply:
Same as above.